docs/resources/namespace_split.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+FEDORA_DATA="${FEDORA_DATA:-/usr/local/fedora/}";
+
+if [[ -z "${NAMESPACE}" ]]; then
```

Comment:
Should we have this as an opt like -n namespace when he runs the script?

Reply:
Exporting the variable prior should be sufficient I think.